src/H5Spoint.c

```diff
@@ -1540,7 +1540,7 @@ H5S__get_select_elem_pointlist(const H5S_t *space, hsize_t startpoint, hsize_t n
             startpoint--;
             node = node->next;
         } /* end while */
-    } /* end else */
```

Comment:
Because we have alignment of comments on. There is no blank line between the two comments, so they are a single block for the formatter. 
Personally, I think the indentation is even less important then the use of "end" comments on short blocks (is the "end while" really necessary?)